### ISSUE TITLE
xSQLServerLogin cannot be used for the same login on multiple SQL server instances on the same server

### ISSUE THREAD

**Issue by nebffa** [PowerShell/xSQLServer#31]
Because the login name is used as the key property for the xSQLServerLogin resource, you can't use multiple xSQLServerLogin resources as there will be key clashes:

```
Resources have identical key properties but there are differences in the following non-key properties:
'SQLInstanceName'. Values 'SECONDINSTANCE' don't match values 'MSSQLSERVER'. Please update these property values so that
they are identical in both cases.
```

**Comment by johlju** [comment]
@nebffa If you have the same login name to the same instance that will not work. Is your scenario that you want to use the same login name to different instances?  

I have two of them in a row, but with different login names, and that works. 

```
        xSQLServerLogin SQLConfigureLoginPrimary
        {
            Name = $SqlServiceCredentialNode1.UserName
            LoginType = 'WindowsUser'
            SQLServer = $Node.NodeName
            SQLInstanceName = $Node.SqlInstanceName

            PsDscRunAsCredential = $SqlAdministratorCredential

            DependsOn = '[Service]SQLServerServiceStarted'
        }

        xSQLServerLogin SQLConfigureLoginSecondary
        {
            Name = $SqlServiceCredentialNode2.UserName
            LoginType = 'WindowsUser'
            SQLServer = $Node.NodeName
            SQLInstanceName = $Node.SqlInstanceName
        
            PsDscRunAsCredential = $SqlAdministratorCredential
        
            DependsOn = '[Service]SQLServerServiceStarted'
        }
```

**Comment by johlju** [comment]
@nebffa Ah, my bad... It was clear stated in the subject that it was to different instances.

**Comment by johlju** [comment]
This will be fixed when PR #106 is merged

**Comment by johlju** [comment]
This was resolved in PR #106

**Comment by johlju** [comment]
@mbreakey3 Can you close this issue? It wasn't auto-closed with the PR apparently.
